src/combobox/example/index.ts

```diff
@@ -142,7 +142,9 @@ export class App extends WidgetBase<WidgetProperties> {
 	}
 
 	render(): DNode {
-		return v('div', [
+		return v('div', {
+			styles: { maxWidth: '256px' }
```

Comment:
@bitpshr I actually wouldn't mind this added to textinput/textarea too, if you feel like it